src/components/@shared/atoms/Container.module.css

```diff
@@ -7,13 +7,13 @@
   width: 100%;
 }
 
-@media (min-width: 40rem) {
+@media (min-width: 55rem) {
   .container {
     padding-left: calc(var(--spacer) / 1.2);
     padding-right: calc(var(--spacer) / 1.2);
   }
 }
 
 .container.narrow {
```

Comment:
it's not narrower than the standard `.container` anymore so this class name is really misleading. Same goes for the prop `narrow` which with this change also became misleading, so prop and class name need to be changed to something describing what we do here